addons/account_edi_peppol/data/peppol_ubl_templates.xml

```diff
@@ -0,0 +1,198 @@
+                <cbc:LineExtensionAmount
+                    t-att-currencyID="invoice.currency_id.name"
+                    t-esc="format_monetary(convert_monetary(line.price_subtotal, line.currency_id or invoice.currency_id))"/>
+                <cac:Item>
+                    <cbc:Description t-esc="line.name.replace('\n', ', ')"/>
+                    <cbc:Name t-esc="line.product_id.name"/>
+                    <cac:SellersItemIdentification t-if="line.product_id.default_code">
+                        <cbc:ID t-esc="line.product_id.default_code"/>
+                    </cac:SellersItemIdentification>
+                    <t t-if="line.tax_ids">
+                        <t t-foreach="line.tax_ids" t-as="tax">
+                            <cac:ClassifiedTaxCategory>
+                                <cbc:ID>S</cbc:ID>
+                                <cbc:Percent t-esc="tax.amount" />
+                                <cac:TaxScheme>
+                                    <cbc:ID>VAT</cbc:ID>
+                                </cac:TaxScheme>
+                            </cac:ClassifiedTaxCategory>
+                        </t>
+                    </t>
+                    <t t-else="" >
+                        <cac:ClassifiedTaxCategory>
+                            <cbc:ID>Z</cbc:ID>
+                            <cbc:Percent>0</cbc:Percent>
+                            <cac:TaxScheme>
+                                <cbc:ID>VAT</cbc:ID>
+                            </cac:TaxScheme>
+                        </cac:ClassifiedTaxCategory>
+                    </t>
+                </cac:Item>
+                <cac:Price>
+                    <cbc:PriceAmount
+                        t-att-currencyID="invoice.currency_id.name"
+                        t-esc="format_monetary(convert_monetary(line.price_subtotal, line.currency_id or invoice.currency_id))"/>
+                    <cbc:BaseQuantity t-esc="line.quantity"/>
+                </cac:Price>
+            </cac:InvoiceLine>
+        </template>
+
+        <!--
+        Render an invoice to be added to the UBL xml document.
+        -->
+        <template id="export_peppol_invoice">
+            <Invoice
+                xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2"
+                xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2"
+                xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2">
+                <cbc:CustomizationID>urn:cen.eu:en16931:2017#compliant#urn:fdc:peppol.eu:2017:poacc:billing:3.0</cbc:CustomizationID>
+                <cbc:ProfileID>urn:fdc:peppol.eu:2017:poacc:billing:01:1.0</cbc:ProfileID>
+                <cbc:ID t-esc="invoice.ref"/>
+                <cbc:IssueDate t-esc="invoice.invoice_date"/>
+                <cbc:DueDate t-if="invoice.invoice_date_due" t-esc="invoice.invoice_date_due.strftime('%Y-%m-%d')" />
```

Comment:
`due_date` is passed inside the dictionary. Use it instead of reformating the date

Reply:
same for `invoice.invoice_date` >< Need a method `format_date` ?